### PR TITLE
server: add persist file log

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?
Issue Number: ref #4399

### What is changed and how it works?
add a log for persisting file.

### Release note

```release-note
None
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

**Review by AndreMouche — APPROVED** [review]
LGTM

File: server/server.go [hunk 1]
```diff
@@ -1464,6 +1464,7 @@ func (s *Server) ReplicateFileToMember(ctx context.Context, member *pdpb.Member,
 
 // PersistFile saves a file in DataDir.
 func (s *Server) PersistFile(name string, data []byte) error {
+	log.Info("persist file", zap.String("name", name), zap.Binary("data", data))
```
Comment (rleungx) [inline server/server.go:1467]: Is it readable?

Comment (disksing) [reply server/server.go:1467]: Not readable. It will use base64. Because the API supports write arbitrary data, binary should be a good choice.

Comment (nolouch) [reply server/server.go:1467]: why need to log the data？may be an md5 or checksum?

Comment (disksing) [reply server/server.go:1467]: It should not be very long. Sometime if we really want to debug, we can decode and check the content.

**Review by nolouch — APPROVED** [review]

**Pull request merged**